node-services/Blog/controller/db.js

```diff
@@ -0,0 +1,87 @@
+var db = require("../database.js");
+var express = require("express");
+
+
+const dbOperations = {
+
+  getAllArticles: async () => {
+
+    try{
+    const articles = await db.collection('articles').get();
+    return { 'responseCode': '1', 'articles': articles};
+    }
+    catch(err)
+    {
+        throw new Error(err);   
+    }
+
+  },  
+  createNewArticle: (article) => {
+    
+    try {
+    var title = article.title;
+    var description = article.description;
+    var markdown = article.markdown; 
+    db.collection("articles").add({
+        Title: title,
+        Description: description,
+        Markdown:markdown,
+        
+    })
+    return { 'responseCode': '1' };
+    }
+    catch(err)
+    {
+        throw new Error(err);
+    }
+  },
+
+ findArticle: async (id) => {
```

Comment:
is calling this async a requirement, since you are waiting for db's response this isn't truly async